CHANGELOG.rst

```diff
@@ -5,32 +5,124 @@ Changelog
 
 0.14.0
 ------
-**Date:** TBA
+**Date:** 2021-06-XX
 
-...
+Resize, Dask, openpmd-pipe and new ADIOS2 Iteration Encoding
 
-...
+This release adds support for resizable data sets.
```

Comment:
User-facing dcumentation to do's for @ax3l and @franzpoeschel:
- [ ] `openpmd-pipe.py` tool should be described in CLI section: https://openpmd-api.readthedocs.io/en/latest/utilities/cli.html
- [ ] the keyword `variableBased` does not occur in the docs (we write variable-based in text in the ADIOS2 section, which makes this hard to search for)
- [ ] we need a user-facing example demonstrating the `Dataset` resize feature